test_1_1/task_1/main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+#include "intPalindrome.h"
+
+int readValue(const char *prompt, const char *incorrectValueMessage) {
+    int value;
```

Comment:
```suggestion
    int value = 0;
```
По стайлгайду любая переменная должна быть инициализирована (и в реальной практике тоже, если переменная не имеет начального значения, то только потому, что профилировщик показал, что его присваивание критично).